paasta_tools/cli/cmds/local_run.py

```diff
@@ -542,7 +542,7 @@ def run_docker_container(
 
         # If the service has a healthcheck, simulate it
         if healthcheck_mode:
-            status = simulate_healthcheck_on_service(
+            status, _ = simulate_healthcheck_on_service(
```

Comment:
Hmm. Makes me wonder why I/we bother to return the string part anyway if the simulate function handles printing it out.